### PR TITLE
Added information from help page to student summary.

### PR DESCRIPTION
**Pull request by reish981**
This page will change based on the values the instructor has saved for an attendance assignment and the attendance records saved for the course

### CODE REVIEW

File: src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp [hunk 1]
```diff
@@ -80,6 +80,96 @@
...
+                <div class="panel-heading" role="tab" id="headingOne">
+                    <h4 class="panel-title">
+                        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne"
+                           aria-expanded="false" aria-controls="collapseOne">
+                            Understanding how your attendance grade is calculated
+                        </a>
+                    </h4>
+                </div>
+                <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+                    <div class="panel-body">
```
Comment (KSU-Github) [inline src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp]: MAJOR Remove this deprecated "border" attribute. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
MINOR Add an "alt" attribute to this image. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)

**Comment by KSU-Github** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/6f0fcb43ef6161eb99c676b062c67b537ccfea00/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by kawesome** [comment]
Can one of the admins verify this patch?

**Comment by KSU-Github** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/26bbad8721b631ef13d4421aba07995b359853dd/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp [hunk 2]
```diff
@@ -80,6 +80,107 @@
...
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100)} * ${assignmentPoints} =
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100) * assignmentPoints}</p>
+
+                        <br>
+                        <p>${attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesTardy * (tardyPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesMissed * (absentPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100) * assignmentPoints} =
+
+                                ${(attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100) * assignmentPoints) +
```
Comment (zoglmannk) [inline src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp]: Please move the calculations into an appropriate Java class where it is possible to unit test them and avoid duplication of the calculations spread across the page.

File: src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java [hunk 3]
```diff
@@ -117,6 +122,14 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
         page.addObject("sectionId", sectionId);
         page.addObject("student", student);
         page.addObject("summaryForm", makeupForm);
+        if (assignment != null){
```
Comment (killsto) [inline src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java]: There should be a space between the ending of the condition and the curly brace which starts the body. If this is happening frequently you might want to check your IDE settings

**Comment by killsto** [comment]
The student summary controller is pretty complex. According to sonar, the cyclomatic complexity of the method is 11. Additionally, the name of the private method isn't descriptive of what the method is doing. We should probably refactor it if there is time.

**Comment by KSU-Github** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/eb80939a8e8dfb04026c9d977e5f1d0478f53fd1/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**